moped-editor/src/utils/activityLogFormatters/mopedFundingActivity.js

```diff
@@ -10,46 +10,61 @@ export const formatFundingActivity = (
   const entryMap = ProjectActivityLogTableMaps["moped_proj_funding"];
 
   const changeIcon = <MonetizationOnOutlinedIcon />;
-  let changeDescription = "Project funding updated";
-  let changeValue = "";
+  const changeText = [];
 
   // add a new funding source
   if (change.description.length === 0) {
-    changeDescription = "Added a new funding source";
     // if the added record has a funding source, use that as the change value
     if (change.record_data.event.data.new.funding_source_id) {
-      changeValue =
-        fundingSources[change.record_data.event.data.new.funding_source_id];
+      changeText.push({ text: "Added ", style: null });
+      changeText.push({
+        text: fundingSources[
+          change.record_data.event.data.new.funding_source_id
+        ],
+        style: "boldText",
+      });
+      changeText.push({ text: " as a new funding source.", style: null });
       // if not, then check if theres a funding program
     } else if (change.record_data.event.data.new.funding_program_id) {
-      changeValue =
-        fundingPrograms[change.record_data.event.data.new.funding_program_id];
+      changeText.push({ text: "Added ", style: null });
+      changeText.push({
+        text: fundingPrograms[
+          change.record_data.event.data.new.funding_program_id
+        ],
+        style: "boldText",
+      });
+      changeText.push({ text: " as a new funding source.", style: null });
+    } else {
+      changeText.push({ text: "Added a new funding source", style: null });
     }
-    // if we have a change value, tack on a : at the end
-    if (changeValue) {
-      changeDescription = "Added a new funding source: ";
-    }
-    // if there isnt a funding source or program added, then default to ""
-    return { changeIcon, changeDescription, changeValue };
+
+    return { changeIcon, changeText };
   }
 
   // delete an existing record
   if (change.description[0].field === "is_deleted") {
-    changeDescription = "Deleted a funding source";
     // if the added record has a funding source, use that as the change value
```

Comment:
should this comment say deleted record instead of added 🤔

Reply:
you are very right thank you!